### ISSUE TITLE
Burst selector: "all" and "none"

### ISSUE THREAD

**Issue by basille** [mablab/rpostgisLT#23]
In the burst selector, it would be nice to have the ability to select all bursts at once, or on the contrary deselect them all. That would make some manipulations a lot quicker with `pgtraj`es with many bursts. Maybe "all" and "none" options in the burst list, or check boxes, or…

**Comment by balazsdukai** [comment]
Yes this feature was there until 4168e55df453d7e13712d348d8038676790862b3 but I removed it, because the widget that has all/none options has a bug. Thus I reverted to the default widget that you see now.

**Comment by basille** [comment]
That's why I remembered such an option. Too bad it's buggy, it would have been nice to have.

**Comment by basille** [comment]
Reopening that one: would it be possible through the use of check boxes, or buttons independent from the widget?

**Comment by balazsdukai** [comment]
Yes, but the optimal solution would be to fix the buggy widget.

**Comment by basille** [comment]
Agreed. Are you volunteering? :D

**Comment by balazsdukai** [comment]
No, but I should at least file a bug report.

**Comment by balazsdukai** [comment]
In shinyWidgets 0.3.4 the `pickerInput` is fixed and now *Deselect all* returns `NULL`.

```
library(shiny)
library(shinyWidgets)

if (interactive()) {

ui <- fluidPage(
  pickerInput(inputId = "somevalue", label = "A label",
              choices = c("a", "b", "c"), multiple = TRUE,
              options = list(`actions-box` = TRUE),),
  verbatimTextOutput("value")
)
server <- function(input, output) {
  output$value <- renderPrint({ input$somevalue })
}
shinyApp(ui, server)
}
```

**Comment by basille** [comment]
Does that mean that the bug has been fixed with the latest update of `shinyWidgets`?

**Comment by balazsdukai** [comment]
Yes and that it needs to be implemented in rpostgisLT.
